distribution/kernel/pom.xml

```diff
@@ -152,6 +152,17 @@
                                     </outputDirectory>
                                     <destFileName>platform-osgi-condpermadmin-${project.version}.jar</destFileName>
                                 </artifactItem>
+                                <artifactItem>
+                                    <groupId>ddf.config</groupId>
+                                    <artifactId>config-external-agent</artifactId>
```

Comment:
Not a big deal right now, but I'm not crazy about putting this in kernel. Kernel should really just be thirdparty code since it's being used as a test container.